server/server.go

```diff
@@ -62,16 +71,85 @@ func (s *Server) Start() {
 	router := mux.NewRouter()
 
 	// Register routes with their corresponding handler funcs.
-	router.HandleFunc("/api/hello", s.helloHandler).Methods("GET")
+	router.HandleFunc("/health", s.healthHandler).Methods("GET")
```

Comment:
```suggestion
	router.HandleFunc("/health", s.healthHandler).Methods(http.MethodGet)
```

A bit nit-picky, but since the standard library defines constants like this, I think it's a better idea to use them than to hard-code this in, even though the constant's value will probably never change, and the code is just as readable without using this constant.